odoo/addons/test_lint/tests/test_jstranslate.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import re
+
+from odoo.modules import get_resource_from_path
+
+from . import lint_case
+
+_logger = logging.getLogger(__name__)
+
+TSTRING_RE = re.compile(r'_l?t\(`.*?`\)', re.DOTALL)
```

Comment:
Might be a good idea to have `\s*` between the parenthesis and the backticks.

Reply:
Thanks for the hint, I changed the regular expression and added a test for your example too.